_includes/pages/confirm-error.html

```diff
@@ -0,0 +1,6 @@
+<div class="container">
+  <div class="page-title">
+    <h1>{{ site.data.translations.confirm-error.title[page.lang] }}</h1>
+    <h2>{{ site.data.translations.confirm-error.subtitle[page.lang] }}</h2>
+  </div>
+</div>
```

Comment:
Could we rather use a layout and markdown rather than an include? 🙂

Reply:
Also wondering if it wouldn't be more maintainable to just create a page for each language rather than using the `data.translations` logic.

Reply:
> Could we rather use a layout and markdown rather than an include? 🙂

We're not going to have so much improvement and we risk putting content in what today corresponds to the titles and subtitles section. I propose that we keep the markdown file mechanism for the content sections

Reply:
> Also wondering if it wouldn't be more maintainable to just create a page for each language rather than using the `data.translations` logic.

Not in favor of this neither as it will split content in two folders and that's exactly what I wanted to avoid to simplify edition for non-tech guys.